pkg/kubectl/cmd/rollout/rollout_status.go

```diff
@@ -62,7 +65,8 @@ func NewCmdRolloutStatus(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	usage := "identifying the resource to get from a server."
 	cmdutil.AddFilenameOptionFlags(cmd, options, usage)
-	cmd.Flags().BoolP("watch", "w", true, "Watch the status of the newest rollout until it's done.")
+	cmd.Flags().BoolP("watch", "w", true, "Watch the status of the rollout until it's done.")
+	cmd.Flags().Int64("revision", 0, "Pin to a specific revision for showing its status. Default to 0 (last revision).")
```

Comment:
Switched to "Defaults". Re 0, we already use it in [rollout history --revision](https://github.com/kubernetes/kubernetes/blob/4bf35b6827cc6f3d33a7bede52edafdfa939a325/pkg/kubectl/cmd/rollout/rollout_history.go#L60) so let's keep it consistent here.